crates/ibc/src/events.rs

```diff
@@ -188,6 +204,57 @@ impl FromStr for IbcEventType {
     }
 }
 
+/// Message event from an IbcEventType
+impl TryFrom<IbcEventType> for abci::Event {
+    type Error = Error;
+
+    fn try_from(event_type: IbcEventType) -> Result<Self, Self::Error> {
+        match event_type {
+            IbcEventType::CreateClient
+            | IbcEventType::UpdateClient
+            | IbcEventType::UpgradeClient
+            | IbcEventType::ClientMisbehaviour => Ok(Self {
+                kind: "message".to_string(),
+                attributes: vec![
+                    ("module", "ibc_client", true).into(),
+                    ("action", event_type.as_str(), true).into(),
+                ],
+            }),
+            IbcEventType::OpenInitConnection
+            | IbcEventType::OpenTryConnection
+            | IbcEventType::OpenAckConnection
+            | IbcEventType::OpenConfirmConnection => Ok(Self {
+                kind: "message".to_string(),
+                attributes: vec![
+                    ("module", "ibc_connection", true).into(),
+                    ("action", event_type.as_str(), true).into(),
```

Comment:
where is this `"action"` attribute in ibc-go? [I only see](https://github.com/cosmos/ibc-go/blob/6784019546e046210c8277ff0d7c3b17628ad9a7/modules/core/04-channel/keeper/events.go#L228) the `"module"` one.

Reply:
I refer to [this spec](https://github.com/cosmos/ibc-go/blob/main/docs/ibc/events.md#sendpacket-application-module-call). But there is a note "This document is unmaintained and may be out of date!".
It's better to be compatible with ibc-go. I'll fix them.